com.sublimetext.three.json

```diff
@@ -1,11 +1,11 @@
 {
     "app-id": "com.sublimetext.three",
     "runtime": "org.freedesktop.Sdk",
-    "runtime-version": "1.6",
+    "runtime-version": "18.08",
     "sdk": "org.freedesktop.Sdk",
     "base": "io.atom.electron.BaseApp",
-    "base-version": "stable",
-    "branch": "stable",
+    "base-version": "18.08",
```

Comment:
I don't see the `18.08` version of `io.atom.electron.BaseApp` in the flathub repository.